app/controllers/api/v0/manifests_controller.rb

```diff
@@ -1,7 +1,8 @@
 class Api::V0::ManifestsController < ApplicationController
   def search
     if !params[:q] && !params[:aq]
-      render json: {}, status: 400
+      manifests = Manifest.all
```

Comment:
this does not make sense to me. An invalid search request returns all the Manifests?